src/rpc/handlers/AccountTx.cpp

```diff
@@ -152,8 +152,19 @@ doAccountTx(Context const& context)
     std::optional<size_t> minReturnedIndex;
     for (auto const& txnPlusMeta : blobs)
     {
-        if (txnPlusMeta.ledgerSequence < minIndex ||
-            txnPlusMeta.ledgerSequence > maxIndex)
+        if (txnPlusMeta.ledgerSequence < minIndex && !forward)
+        {
+            response.erase(JS(marker));
+            break;
+        }
+
+        if (txnPlusMeta.ledgerSequence > maxIndex && forward)
+        {
+            response.erase(JS(marker));
+            break;
+        }
```

Comment:
We could combine this logic into one if statement, since the body of each is the same